kubeadm/scripts/README.md

```diff
@@ -0,0 +1,4 @@
+# Warning
+
+The instructions and scripts in the directory DO NOT configure a CNI solution for Windows nodes running containerd.
+There is a work-in-progress PR to assist in this at https://github.com/kubernetes-sigs/sig-windows-tools/pull/239
```

Comment:
can we add this warning to https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/guides/guide-for-adding-windows-node.md as well?  I think this is the one folks land on too for HPC

Reply:
Yes I was looking for that page too but didn't see it :(